manager/controllers/app/m4dapplication_controller.go

```diff
@@ -403,7 +403,11 @@ func AnalyzeError(app *app.M4DApplication, log logr.Logger, assetID string, err
 }
 
 func ownerLabels(id types.NamespacedName) map[string]string {
-	return map[string]string{OwnerLabelKey: id.Namespace + "." + id.Name}
+	return map[string]string{
+		OwnerLabelKey: id.Namespace + "." + id.Name,
```

Comment:
Adding other labels makes OwnerLabelKey redundant, but ok - I'll handle it later.